### PR TITLE
Revert "fix caching"

### PR DESCRIPTION
**Pull request by peterbe**
Reverts mdn/translated-content#114

First, let's do this... Then I'm going to see if I can solve the actual bug.

### CODE REVIEW

**Comment by peterbe** [comment]
Ok @fiji-flo now it was able to skip the `yarn install` and benefit from its cache: https://github.com/mdn/translated-content/pull/121/checks?check_run_id=2097260750 ...without breaking.

File: .github/workflows/pr-test.yml [hunk 1]
```diff
@@ -33,19 +28,16 @@ jobs:
...
+      - name: Cache node_modules
+        uses: actions/cache@v2.1.4
+        id: cached-node_modules
         with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          path: |
+            ${{ github.workspace }}/mdn/content/node_modules
```
Comment (peterbe) [inline .github/workflows/pr-test.yml:36]: This used to be:
```
              {{ github.workspace }}/mdn/content/node_modules
```
(note the missing `$` at the start)
That was probably the bug.

Comment (fiji-flo) [reply .github/workflows/pr-test.yml:36]: 🤞

**Review by fiji-flo — APPROVED** [review]
Thanks

**Pull request merged**